mrbgems/mruby-bigint/core/bigint.h

```diff
@@ -54,10 +56,82 @@ typedef struct _mpz_t {
 
 struct RBigint {
   MRB_OBJECT_HEADER;
-  mpz_t mp;
+  union {
+    mpz_t heap;
+    mp_limb ary[RBIGINT_EMBED_SIZE_MAX];
+  } as;
 };
 #define RBIGINT(v) ((struct RBigint*)mrb_ptr(v))
 
+/*
+ *  flags of struct RBigint
+ *
+ *  6..:  UNUSED
+ *  4..5: sign flags
+ *        00: negative  (<--> -1)
+ *        01: zero      (<-->  0)
+ *        10: positive  (<--> +1)
+ *        11: UNUSED
+ *  0..3: size of embeded array; 15 means used with heap
```

Comment:
```suggestion
 *  0..3: size of embedded array; 15 means used with heap
```

Reply:
Thanks, I'll fix the Fuzzing error and then apply it.
It may take some time.......